db/migrate/20220112230653_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :email
+      t.string :password_digest
+      t.boolean :is_admin
```

Comment:
In this case it would be correct to validate the same thing you are validating in the model:
```suggestion
      t.string :name, null: false
      t.string :email, unique: true
      t.string :password_digest, null: false
      t.boolean :is_admin, default: false
```